run_simulator.sh

```diff
@@ -3,23 +3,44 @@
 date
 set -x
 
+echo "ROUND_START: ${ROUND_START:=1}"
+echo "ROUND_END: ${ROUND_END:=1}"
 echo "NUMBER_OF_SERVERS: ${NUMBER_OF_SERVERS:=1}"
 echo "DURATION: ${DURATION:=120}"
 echo "LOAD: ${LOAD:=$((${NUMBER_OF_SERVERS} * 80))}"
-echo "SCENARIO: ${SCENARIO}"
-RESULTS_PATH="${SCENARIO}${NUMBER_OF_SERVERS}instance(s)"
-echo "OUTPUT_PATH: ${OUTPUT_PATH:=/tmp/simulation_results/}"
 
+if [ "${NUMBER_OF_SERVERS}" == "1" ];
```

Comment:
Do we really need proper English to make directory names?

In my experiments, I am using 1i, 2i, 4i and so on. Wouldn't that be enough? Bear in mind that unnecessary complexity had already caused problems.

Reply:
It seems better to me. Again, thanks!